client/src/__tests__/AppRoutes.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import { Redirect, MemoryRouter } from 'react-router';
+import AppRoutes from '../components/routes';
+import Index from '../components/index/index';
+import Create from '../components/createBlog/create';
+import Adapter from 'enzyme-adapter-react-16';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+
+configure({ adapter: new Adapter() });
+const initialState = { auth: { isLoggedIn: true } }
+
+describe('Routes tests', () => {
+  const mockStore = configureStore();
+  let store, wrapper;
+
+  beforeEach(() => {
+    store = mockStore(initialState)
+  })
+
+  it('Index route', () => {
+
+    wrapper = mount(
```

Comment:
Why do you use `mount`, tell me pls?
It will be better use `shallow` without a mocking store.

Reply:
Through this case i`m testing react routing, bot rendering component.